### ISSUE TITLE
Bug with UserPassBaseLimiter

### ISSUE THREAD

**Issue by tvdijen** [cirrusidentity/simplesamlphp-module-ratelimit#28]
Oct 26 11:36:10 IDP[6695]: 3 [f1b9757f3e] Caused by: TypeError: Return value of SimpleSAML\Module\ratelimit\Limiters\UserPassBaseLimiter::getCurrentCount() must be of the type int, object returned

To me, this only seems possible if an object was _stored_ in the first place, but I don't see how that could happen. The store used is `memcached`.

**Comment by tvdijen** [comment]
I found out that the object is of type `SimpleSAML\Session`...

**Comment by pradtke** [comment]
That is perplexing.  The session storage stuff puts a `.session.` in its key names and rate limiting puts a `.int.ratelimit` in its key names. It seems unlikely that they would both generate an identical key.  I'm not sure where that leaves us for figuring out the cause.

**Comment by tvdijen** [comment]
Yeah, it's really weird.. I haven't been able to figure it out.. Restarting memcached solved the problem and I haven't seen this error since, so there must have been some kind of interference...
